test/test_allkinds.py

```diff
@@ -1523,14 +1523,30 @@ def test_styles():
 
 
 def test_né():
+    # Attn. error codes 'P_Né' and 'P_NT_Né' are used,
+    # depending on where the error is handled
     s = "Ég fór út né gekk heim."
     check_sentence(s, [(3, 3, "P_NT_Né")])
     s = "Ég hvorki fór út né gekk heim."
     check_sentence(s, [])
     s = "Við keyptum brauð né ost."
-    check_sentence(s, [(3, 3, "P_NT_Né")])
+    check_sentence(s, [(3, 3, "P_Né")])
     s = "Við keyptum annaðhvort brauð né ost."
-    check_sentence(s, [(4, 4, "P_NT_Né")])
+    check_sentence(s, [(4, 4, "P_Né")])
+    s = "Ég hugsa ekki um hvað Bretland gerir né hvað Ísland gerir."
+    check_sentence(s, [(7, 7, "P_NT_Né")])
```

Comment:
Is there a difference between P_NT_Né and P_Né?

Reply:
See comment above.